src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -660,9 +663,19 @@ namespace OpenMS
       if (cexp_->getInstrument() != Instrument() || cexp_->getContacts().size() != 0)
       {
         const Instrument& inst = cexp_->getInstrument();
+        // the Instrument Manufacturer is paramount for some downstream tools
+        // Since the .getVendor() is usually empty, we infer this via the Aquisiton Software, which is unique to Thermo
+        String manufacturer = inst.getVendor();
+        if (String(inst.getSoftware().getName()).toLower().hasSubstring("xcalibur"))
```

Comment:
hmm ok, sounds fine to me if MQ needs this. I just hope it wont create problems in the future...